modules/farm_fd2/src/entrypoints/soil_disturbance/lib.submitError.unit.cy.js

```diff
@@ -167,163 +172,165 @@ function runTest(activePlantAsset) {
       cleanupLogsAndAssets(movementLogs, plantAssets);
     });
 
-    it(
-      'Soil Disturbance: records are deleted if there is a submission error',
-      { retries: 0 },
-      () => {
-        if (activePlantAsset) {
-          // Counter to track the number of activity logs
-          let postRequestCount = 0;
-
-          cy.intercept('POST', '**/api/log/activity', (req) => {
-            postRequestCount += 1;
-            if (postRequestCount === 3) {
-              req.reply({
-                statusCode: 401,
-              });
-            } else {
-              // Continue with the request normally for other requests
-              req.continue();
-            }
-          }).as('actPost');
+    it('Soil Disturbance: records are deleted if there is a submission error', () => {
+      //{ retries: 4 },
+      if (activePlantAsset) {
+        // Counter to track the number of activity logs
+        let postRequestCount = 0;
 
-          let standardQuantityDeletes = 0;
-          cy.intercept('DELETE', '**/api/quantity/standard/*', (req) => {
-            standardQuantityDeletes++;
+        cy.intercept('POST', '**/api/log/activity', (req) => {
+          postRequestCount += 1;
+          if (postRequestCount === 3) {
+            req.reply({
+              statusCode: 401,
+            });
+          } else {
+            // Continue with the request normally for other requests
             req.continue();
-          }).as('quantDelete');
+          }
+        });
 
-          let activityLogDeletes = 0;
-          cy.intercept('DELETE', '**/api/log/activity/*', (req) => {
-            activityLogDeletes++;
-            req.continue();
```

Comment:
Should we still be failing these deletes so that we get the error messages here? That would be consistent with what is done in the else for the no active plant asset case.

Reply:
If we fail these deletes, then there will be logs in the DB that will be referencing the plant asset that was created just for the test.  The test (in the `after`) then attempts to delete the plant asset that was created for the test, but this will fail because the logs created by the soil disturbance will refer to it.  So not failing these deletes ensure that the logs created by the soil disturbance will be deleted, and thus the `after` will be able to delete the plant asset created for the test.